cluster/common.sh

```diff
@@ -99,7 +114,10 @@ function create-kubeconfig() {
     "${kubectl}" config set-credentials "${CONTEXT}" "${user_args[@]}"
   fi
   "${kubectl}" config set-context "${CONTEXT}" --cluster="${CONTEXT}" --user="${CONTEXT}"
-  "${kubectl}" config use-context "${CONTEXT}"  --cluster="${CONTEXT}"
+
+  if [[ "${SECONDARY_KUBECONFIG}" != "true" ]];then
```

Comment:
I want the semantics as follows: 

`To skip setting this kubeconfig as the current-context, set SECONDARY_KUBECONFIG=true. Otherwise, the new context will be set as the current-context`